style.css

```diff
@@ -1,28 +1,68 @@
 
 
-//////////////////////////////////////////
-div {
-  background-color: black;
+nav li{
+  display: inline;
+  margin-left: auto;
+}
+nav ul{
+    padding: 10px;
+    background-color: #973124;
+}
+div{
+  border: 2px solid;
+  border-color: #973124;
+  width: 50%;
+  height:auto;
+  margin-left: auto;
+  margin-right: auto;
+
+}
+*{
```

Comment:
General selectors like this should probably go at the top of the file so that you can override them later if you want.